static/cross.svg

```diff
@@ -0,0 +1,14 @@
+<svg
```

Comment:
The icon looks nice, but I wonder does the currently installed fontawesome library provide close icon we can use? I believe it could be used